### PR TITLE
Update

### PR DESCRIPTION
**Pull request by Sunaries**
Overall improvement to text.

### CODE REVIEW

**Comment by Sunaries** [comment]
Good to merge.

File: EnglishTranslation-master/ETC.tsv [hunk 1]
```diff
@@ -21778,7 +21778,7 @@ ETC_20160520_021777	Collection: Never Seen Before
 ETC_20160520_021778	Ash Blonde Dye
 ETC_20160520_021779	A subtle golden color is the best to attract attention. Right-click to use. Check your Character Info for the registered color.
 ETC_20160520_021780	Ruby Wine Dye
-ETC_20160520_021781	A fiery and fierce red like ruby. Right-click to use. Check your Character Info for the registered color.
+ETC_20160520_021781	Fiery and fierce red like ruby. Right-click to use. Check your Character Info for the registered color.
```
Comment (sunhwapark) [inline EnglishTranslation-master/ETC.tsv:21781]: Character info > Character Settings

Context: 
UI_20151224_001964
UI_20150323_001369

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Pull request merged**